src/test/kotlin/uk/gov/justice/digital/hmpps/makerecalldecisionapi/MrdTestDataBuilder.kt

```diff
@@ -486,6 +489,44 @@ class MrdTestDataBuilder {
       ),
     )
 
+    fun buildCvlLicenceConditionsBreached(
+      useTransformedSelected: Boolean = false,
```

Comment:
Could you add a comment explaining this function? Mostly useTransformedSelected as it's not totally clear why you would/wouldn't use it e.g. 

```
/**
 * Builds a set of licence conditions that have been breached
 *
 * @param useTransformedSelected
 *  -  If false (default), returns the uuid for the breached conditions
 *  -  If true, mocks transformConditionSection and returns the description of the breached conditions
 *
 */
```